core/templates/dev/head/pages/exploration_editor/editor_tab/TrainingDataService.js

```diff
@@ -82,28 +79,23 @@ oppia.factory('TrainingDataService', [
         ResponsesService.updateConfirmedUnclassifiedAnswers(
           confirmedUnclassifiedAnswers);
       }
-
-      var index = _removeAnswerFromTrainingData(answer, _trainingDataAnswers);
-      if (index !== -1) {
-        _trainingDataFrequencies.splice(index, 1);
-        $rootScope.$broadcast('updatedTrainingData');
-      }
     };
 
     return {
-      initializeTrainingData: function(explorationId, stateName) {
+      getTrainingDataAnswers: function() {
+        var trainingDataAnswers = [];
         var answerGroups = ResponsesService.getAnswerGroups();
+
+        // Remove the answer from all answer groups.
```

Comment:
This doesn't seem to be removing anything?

Reply:
My bad. Will remove it.

Reply:
Done.